.github/workflows/issue-lock.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Lock new issues'
+
+on:
+  issues:
+    types: opened
+
+permissions:
+  issues: write
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
```

Comment:
let's add `if: github.repository == 'reactplay/react-play'` 
to prevent this running on forked repos